### PR TITLE
Tests

### LINKED ISSUE #25: Implement Integration Test(Supertest)
implement supertest testing basic endpoint in the frontend.

### PR DESCRIPTION
**Pull request by Jonas-K-S**
Wrote a lot of tests, apparently we only need 2 tests. One post and one get for backend acceptance tests. I wrote like 23 so maybe we don't need the image test I just wanted some better coverage average.

### CODE REVIEW

File: .DS_Store
Comment (dltompki) [inline .DS_Store:1]: this is a metadata file for the mac filesystem. please remove from PR and add to .gitignore

**Review by dltompki — CHANGES_REQUESTED** [review]
Add screenshot of local test results, since they aren't working in CI.

**Comment by Jonas-K-S** [comment]
Screenshot 2024-06-07 at 11 22 29 PM

File: .DS_Store
Comment (Jonas-K-S) [reply .DS_Store:1]: I don't see it on my end. i added it to the git ignore and there isn't an option for me to delete it here.

Comment (dltompki) [reply .DS_Store:1]: I removed it

**Review by dltompki — APPROVED** [review]

**Pull request merged**